pkg/core/client/client.go

```diff
@@ -52,6 +54,16 @@ func (x NodeInfo) AddressGroup() network.AddressGroup {
 	return x.addrGroup
 }
 
+// SetExternalAddressGroup sets a group of network addresses.
```

Comment:
should it have _the same_ comment as `SetAddressGroup`?

Reply:
Fixed